### PR TITLE
[Core] Fix error log on initialization

### PR DESCRIPTION
**Pull request by Tankilevitch**
# Description

What - Removed error log when looking for integration on initialization

Before

Screenshot 2024-08-29 at 15 46 38

After

Screenshot 2024-08-29 at 15 47 24

Why -

How -

## Type of change

Please leave one option from the following and delete the rest:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] New Integration (non-breaking change which adds a new integration)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Non-breaking change (fix of existing functionality that will not change current behavior)
- [ ] Documentation (added/updated documentation)

<h4> All tests should be run against the port production environment(using a testing org). </h4>

### Core testing checklist

- [ ] Integration able to create all default resources from scratch
- [ ] Resync finishes successfully
- [ ] Resync able to create entities
- [ ] Resync able to update entities
- [ ] Resync able to detect and delete entities
- [ ] Scheduled resync able to abort existing resync and start a new one
- [ ] Tested with at least 2 integrations from scratch
- [ ] Tested with Kafka and Polling event listeners

### Integration testing checklist

- [ ] Integration able to create all default resources from scratch
- [ ] Resync able to create entities
- [ ] Resync able to update entities
- [ ] Resync able to detect and delete entities
- [ ] Resync finishes successfully
- [ ] If new resource kind is added or updated in the integration, add example raw data, mapping and expected result to the `examples` folder in the integration directory. 
- [ ] If resource kind is updated, run the integration with the example data and check if the expected result is achieved
- [ ] If new resource kind is added or updated, validate that live-events for that resource are working as expected
- [ ] Docs PR link [here](#)

### Preflight checklist

- [ ] Handled rate limiting
- [ ] Handled pagination
- [ ] Implemented the code in async
- [ ] Support Multi account

## Screenshots

Include screenshots from your environment showing how the resources of the integration will look.

## API Documentation

Provide links to the API documentation used for this integration.

### CODE REVIEW

**Review by matan84 — APPROVED** [review]
LGTM

**Review by erikzaadi — APPROVED** [review]
This small thing has been confusing as 🦆

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,13 +7,20 @@ this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
...
 - Test support and helpers
 
 
+## 0.10.5 (2024-08-29)
+
+### Bug Fixes
+
+- Fixed error log when looking for existence of integration on initialization
+
+
```
Comment (Tankilevitch) [inline CHANGELOG.md:23]:
```suggestion

## 0.10.6 (2024-08-31)

### Bug Fixes

- Fixed error log when looking for existence of integration on initialization


## 0.10.5 (2024-08-29)

### Improvements

- Test support and helpers


```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,7 +7,14 @@ this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
...
 
-## 0.10.5 (2024-08-27)
+## 0.10.6 (2024-08-31)
+
+### Bug Fixes
+
+- Fixed error log when looking for existence of integration on initialization
+
+
+## 0.10.5 (2024-08-29)
```
Comment (Tankilevitch) [inline CHANGELOG.md:17]:
```suggestion
## 0.10.5 (2024-08-27)
```

**Pull request merged**